components/github/README.md

```diff
@@ -1,257 +1,12 @@
-# Github API Integration Platform
+# Getting Started
 
-### Connect Github + 1000s of apps, remarkably fast.
+## Github Triggers: Webhooks vs. Polling
+The Github triggers are built in way that configures the trigger as a webhook if you are the admin of a repo, and falls back to polling if you are not the admin of a repo.
```

Comment:
```suggestion
The Github triggers in Pipedream enable you to get notified immediately via a webhook if you have `admin` rights on the repo you're watching. Otherwise you can still poll for updates at a regular interval for any other repo where you might not have `admin` rights.
```

Reply:
Doesn't need to be this language exactly, but I think yours could use a little more context for users who might not be familiar with Pipedream.